backend/store/migration/prod/LATEST.sql

```diff
@@ -21,11 +21,11 @@ CREATE TABLE principal (
     type TEXT NOT NULL CHECK (type IN ('END_USER', 'SYSTEM_BOT', 'SERVICE_ACCOUNT')),
     name TEXT NOT NULL,
     email TEXT NOT NULL,
-    password_hash TEXT NOT NULL
+    password_hash TEXT NOT NULL,
+    idp_id INTEGER REFERENCES idp (id),
```

Comment:
Where is the corresponding migration script for ALTER statements under 1.12 folder?

Reply:
The `backend/store/migration/prod/1.12/0005##idp.sql` contains creating idp table and alter principal table.